conduct/artifacts/CODE_OF_CONDUCT_PREFIX.md

```diff
@@ -0,0 +1,44 @@
+# Code of Conduct
+
+The OpenJS Foundation and its member projects use Contributor Covenant as their code of conduct. The full text is included below in English, and [translations](https://www.contributor-covenant.org/translations) are available on the Contributor Covenant website.
+
+## Commitment
+
+All recipients of reports commit to:
+
+- maintain the confidentiality with regard to the reporter and victim of an incident, and
+- participate in the path for escalation as outlined in the section on Escalation when required.
+
+## Report an issue in a project
+
+1. To report an issue in one of the projects listed below, please use the method provided. The project itself is responsible for managing these reports.
+    * **AMP Project:** <code-of-conduct@amp.dev>
+    * **Appium:** email maintainers
+    * **Electron:** <coc@electronjs.org>
+    * **Express.js:** <express-coc@lists.openjsf.org>
+    * **Fastify:** <hello@matteocollina.com> or <tommydelved@gmail.com>
+    * **HospitalRun:** <hello@hospitalrun.io>
+    * **LoopBack** <tsc@loopback.io>
+    * **Node.js:** <report@nodejs.org>
+    * **Node-RED:** <team@nodered.org>
+    * **Webdriver.io:** [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
+    * **Webhint:** <support@webhint.io>
+2. For every other OpenJS Foundation project, please email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
+
+
+## Report an issue in a space managed by the foundation
+
+For reporting issues in spaces managed by the OpenJS Foundation, for example, repositories within the OpenJS organization or an live event such as a conferences, email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
+
+## Escalate an issue
+
+The OpenJS Foundation maintains a [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md#code-of-conduct-panel). This is a foundation-wide team established to manage escalation when a reporter believes that a report to a member project or the CPC has not been properly handled.
```

Comment:
Potentially want to clarify here that this isn't meant to duplicate the above section on reporting an issue in a project. On first read, that wasn't clear to me and I've been involved in this stuff for a long time - I imagine it might be even less clear to people who aren't familiar with this.

Reply:
Wait, the CoCP should handle both reports for projects who don't have their own process AND escalations.

Reply:
The recommendation was ambiguous (for a non-native speaker like me). If you're concerned and have a specific recommendation, can you add this to this review, or can we do it in a different pull request?

Reply:
this is unresolved but shouldn't block merging.